src/PowerShellEditorServices/Extensions/IEditorOperations.cs

```diff
@@ -54,6 +54,13 @@ public interface IEditorOperations
         /// <returns>A Task that can be tracked for completion.</returns>
         Task CloseFile(string filePath);
 
+        /// <summary>
+        /// Causes a file to be saved in the editor.
+        /// </summary>
+        /// <param name="filePath">The path of the file to be saved.</param>
+        /// <returns>A Task that can be tracked for completion.</returns>
+        Task SaveFile(string filePath);
+
```

Comment:
Would adding to this interface be a breaking change?  I know we have at least two new implementations of PSES since this interface was last changed.  Should new features be added as additional interfaces, something like `IEditorSaveOperations`?

Reply:
Good point.. but I think if we were going to make a new interface, I'd want it to be `IEditorFileOperations` and it would contain `NewFile`, `OpenFile`, `CloseFile`, SaveFile`, ya know? And that would surely warrant a breaking change.

Reply:
As far as I know, nobody is using these interfaces outside of PSES itself.  Adding a method to an interface wouldn't be a breaking change so I think a new interface shouldn't be necessary for this.

Reply:
David addressed this in his comment below :)